### DISCUSSION TITLE
M2o Dj Tv Italy

### DISCUSSION THREAD

**Discussion by Vartotojas1234** [iptv-org/community#32] [General]
Please insert M2o Dj Tv Italy. Adress is https://4c4b867c89244861ac216426883d1ad0.msvdn.net/live/S62628868/uhdWBlkC1AoO/playlist.m3u8

**Comment by Vartotojas1234** [comment]
I still can't find the M2o Dj TV channel in the Music folder. Put it there please.

**Comment by Alstruit** [comment]
Please see #6737.
